jenkins-build-pack-api/Jenkinsfile

```diff
@@ -707,6 +785,10 @@ def loadBuildModules(buildModuleUrl){
 		configLoader = configModule.initialize(resultJsonString)
 		echo "config loader loaded successfully."
 
+		awsModule = load "aws-apigateway.groovy"
```

Comment:
rename this to awsApiGatewayModule?

Reply:
why? that is not the standard we are following.

Reply:
@svidhani This module contains functionality related to AWS API Gateway not AWS in general, hence the suggested name to the variable: awsApiGatewayModule (instead of awsModule).

Reply:
renamed to aws-apigateway-module.groovy as per the standard we are following